requirements.txt

```diff
@@ -0,0 +1,15 @@
+jax 
+jaxlib
+flax
+torch
+transformers
+diffusers
+numpy
+optax
+torchvision
+datasets
```

Comment:
I think we'll also want to add `webdataset`, but I can do that when we update the training script.